archaius2-core/src/main/java/com/netflix/archaius/config/PropertiesConfig.java

```diff
@@ -0,0 +1,23 @@
+package com.netflix.archaius.config;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class PropertiesConfig extends MapConfig {
+
+    public static PropertiesConfig fromFile(String filename) throws IOException {
```

Comment:
Based on the name, I would expect this to load from the file system rather than a class loader. Perhaps change to `fromResource` instead?

It would also be useful to have a comment indicating the behavior if multiple resources with the provided name are in the classpath. For example, in archaius v1 the helpers would use `getResources` and load all of the matching property files. In this case it only gets the first one that is found.